extensions/notebook/src/book/bookVersionHandler.ts

```diff
@@ -56,6 +56,15 @@ export function convertFrom(version: string, section: JupyterBookSection): Jupyt
 	}
 }
 
+function setNotNumbered(section: JupyterBookSection): boolean | undefined {
+	if (section.numbered !== undefined) {
```

Comment:
This could be simplified to 

`return section.not_numbered !== undefined ? !section_not_numbered : undefined;`